NEWS.md

```diff
@@ -1,11 +1,32 @@
-# In progress:
-* Installation of local snap components
```

Comment:
I think we haven't completed this yet, so the in progress part likely needs to be restored